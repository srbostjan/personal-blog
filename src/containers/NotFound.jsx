import React, { useEffect } from 'react';
import { Link } from 'react-router-dom';
import ReactGA from 'react-ga';
import { Helmet } from 'react-helmet';

function NotFound() {
  useEffect(() => {
    ReactGA.pageview(window.location.pathname + window.location.search);
  });
  return (
    <div className="not-found">
      <Helmet>
        <title>Not Found</title>
        <meta name="description" content="Page not found" />
      </Helmet>
      <h1>404</h1>
      <p>Page not found</p>
      <Link to="/">Go to homepage</Link>
    </div>
  );
}

export default NotFound;
