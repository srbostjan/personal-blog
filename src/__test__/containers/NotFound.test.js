import React from 'react';
import { mount, shallow } from 'enzyme';
import { create } from 'react-test-renderer';
import NotFound from '../../containers/NotFound';

describe('<NotFound />', () => {
  test('Render componente NotFound', () => {
    const card = shallow(<NotFound />);
    expect(card.length).toEqual(1);
  });
});
describe('NotFound Snapshot', () => {
  test('NotFound Snapshot', () => {
    const component = create(<NotFound />);
    expect(component.toJSON()).toMatchSnapshot();
  });
})