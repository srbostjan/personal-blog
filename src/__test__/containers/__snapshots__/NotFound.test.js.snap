// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Snapshot NotFound Snapshot 1`] = `
<div
  className="not-found"
>
  <h1>
    404
  </h1>
  <p>
    Page not found
  </p>
</div>
`;
